spring-batch-docs/asciidoc/job.adoc

```diff
@@ -58,8 +58,8 @@ declarative flow control (`Decision`) and externalization of flow definitions (`
 [role="xmlContent"]
 There are multiple implementations of the <<job.adoc#configureJob,`Job`>> interface, however, the namespace
 abstracts away the differences in configuration. It has only three
-required dependencies: a name, `JobRepository` , and
-a list of `Step` s.
+required dependencies: a name, a `JobRepository` , and
+a list of ``Step``s.
```

Comment:
For some reason, asciidoctor does not render correctly the following: `Step`s . I found that double backticks fix the rendering issue. You may see this in other places of the PRs.

Reply:
The s after the back tick is a problem. Michael's suggestion is the way to go, provided "instances" is the right word.

Reply:
ok I will reword this accordingly